### DISCUSSION TITLE
Are there any examples with the new VacuumGripper?

### DISCUSSION THREAD

**Discussion by thephet** [cyberbotics/webots#6476] [General]
Hi there. I have an Epson SCARA T6 robot, and I have been using a lot your example. It is called "industrial_example" and a Scara robot picks fruits. As you know in this example the picking operation is fake. One of the fruits magically appears at the end of the vacuum gripper, and then it drops it in one of the boxes.

I have seen that in your last update you have introduced a VacuumGripper. I wonder if there's any example about this, so that I can use it in the Epson T6 example, so that it picks fruits in a more real way.

Thank you for your help.

**Comment by omichel** [comment]
Yes, it's [there](https://github.com/cyberbotics/webots/blob/master/projects/samples/devices/worlds/vacuum_gripper.wbt) (if you search for vacuum in the `File / Open Sample World...` menu of Webots, you will find it as well).

**Reply by thephet** [reply]
Thank you, I can see it. Although I find it a bit complex. I have been navigating the Scene Tree, but I cannot see or find where the Vacuum is.

Also I can see it is in C, is there an example in Python? Would it be possible?

**Reply by omichel** [reply]
It's here: https://github.com/cyberbotics/webots/blob/26e08bbf6049f8d3ec3af3dd874e5f5d4062ab96/projects/samples/devices/worlds/vacuum_gripper.wbt#L98

Yes, the C controller could be translated to Python fairly easily.
